Cargo.toml

```
@@ -12,12 +12,12 @@ repository = "https://github.com/FabianLars/tauri-plugin-oauth"
 links = "tauri-plugin-oauth"
 
 [dependencies]
-tauri = { version = "2.0.0-rc.10" }
+tauri = { version = "2.0.6" }
```

```suggestion
tauri = { version = "2" }
```
i don't wanna force the minor/patch versions unless we really have to